javascript/javascript2/week1/spiritanimal.js

```diff
@@ -0,0 +1,28 @@
+const spiritAnimas = [ "fullmoonwolf","The Butterfly","Teen wolf","Spidersense"]
+const button1 = document.getElementById("btn1");
+
+ const randomNumber1 = Math.floor(Math.random()*10); 
```

Comment:
Instead of using number 10, you need to multiple Math.random() (which can return number in range from 0 to 1) by number of items from your array, which is 4. But you might write it less hardcoded (based on array length).